cellprofiler/knime_bridge.py

```diff
@@ -54,6 +53,8 @@ def ZmqFrame(data=""):
 CLEAN_PIPELINE_REQ_1 = "clean-pipeline-request-1"
 CLEAN_PIPELINE_REPLY_1 = "clean-pipeline-reply-1"
 
+LOGGER = logging.getLogger(__name__)
+
 
 class KnimeBridgeServer(threading.Thread):
```

Comment:
I'm tempted to say let's just comment all this out until we deal with #4689 but that doesn't necessarily have to happen here. But since this removes the javabridge functionality without replacing it with anything, I feel like we can pretty confidently say if the bridge wasn't ALREADY broken (which I'm pretty sure it was), it's definitely likely to be now.